verify-generated-files-remake.sh fails to fail

```
Verifying verify-generated-files-remake.sh
+++ Running case: verify.generated-files-remake 
+++ working dir: /home/prow/go/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-generated-files-remake.sh"
make[2]: *** [Makefile.generated_files:591: pkg/generated/openapi/zz_generated.openapi.go] Error 1
make[1]: *** [Makefile:544: generated_files] Error 2
+++ exit code: 0
[0;32mSUCCESS[0m  verify-generated-files-remake.sh	1613s
```
/sig testing
/sig release

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/92474/pull-kubernetes-files-remake/1275829687804760065

--- a/hack/verify-generated-files-remake.sh
+++ b/hack/verify-generated-files-remake.sh
@@ -338,7 +338,7 @@ assert_clean
 touch "staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go"
 echo > api/api-rules/violation_exceptions.list
 echo > api/api-rules/codegen_violation_exceptions.list
-if make generated_files >/dev/null; then
+if make generated_files >/dev/null 2>&1; then
     echo "Expected make generated_files to fail with API violations."
     echo ""
     exit 1

